### PR TITLE
Fix merge conflicts with v1.13.2 template

### PR DESCRIPTION
**Pull request by ErikDanielsson**
## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/smrnaseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/smrnaseq)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/smrnaseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by ewels** [comment]
Not quite sure what's going on with the linting logs - tonnes of sqlite log messages being spat out 👀 Do you see this locally as well @ErikDanielsson ?

**Comment by ewels** [comment]
## `nf-core lint` overall result: Passed :white_check_mark: :warning:

Posted for pipeline commit 1ab5a5c

```diff
+| ✅ 159 tests passed       |+
!| ❗  24 tests had warnings |!
```

<details>

### :heavy_exclamation_mark: Test warnings:

* [actions_awsfulltest](https://nf-co.re/tools-docs/lint_tests/actions_awsfulltest.html) - `.github/workflows/awsfulltest.yml` should test full datasets, not `-profile test`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::r-base=3.6.3`, `4.0.3` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::r-statmod=1.4.32`, `1.4.35` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::r-data.table=1.12.4`, `1.14.0` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::r-gplots=3.0.1.1`, `3.1.1` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::r-r.methodss3=1.7.1`, `1.8.1` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::python=3.7.3`, `3.9.2` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::markdown=3.1.1`, `3.3.4` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::pymdown-extensions=6.0`, `8.1.1` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::pygments=2.6.1`, `2.8.1` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `conda-forge::pigz=2.3.4`, `2.6` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::trim-galore=0.6.5`, `0.6.6` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::samtools=1.9`, `1.12` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::bowtie=1.2.3`, `1.3.0` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::multiqc=1.9`, `1.10.1` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::htseq=0.11.3`, `0.13.5` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::seqkit=0.12.0`, `0.15.0` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::bioconductor-edger=3.26.5`, `3.32.1` available
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep outdated: `bioconda::bioconductor-limma=3.40.2`, `3.46.0` available
* [pipeline_todos](https://nf-co.re/tools-docs/lint_tests/pipeline_todos.html) - TODO string in `README.md`: _Add bibliography of tools and data used in your pipeline_
* [pipeline_todos](https://nf-co.re/tools-docs/lint_tests/pipeline_todos.html) - TODO string in `nextflow.config`: _Specify your pipeline's command line flags_
* [pipeline_todos](https://nf-co.re/tools-docs/lint_tests/pipeline_todos.html) - TODO string in `awstest.yml`: _You can customise CI pipeline run tests as required_
* [pipeline_todos](https://nf-co.re/tools-docs/lint_tests/pipeline_todos.html) - TODO string in `awsfulltest.yml`: _You can customise AWS full pipeline tests as required_
* [schema_params](https://nf-co.re/tools-docs/lint_tests/schema_params.html) - Schema param `config_profile_name` not found from nextflow config

### :white_check_mark: Tests passed:

* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.gitattributes`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `CHANGELOG.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `CODE_OF_CONDUCT.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `CODE_OF_CONDUCT.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `LICENSE` or `LICENSE.md` or `LICENCE` or `LICENCE.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `nextflow_schema.json`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `nextflow.config`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `README.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/.dockstore.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/CONTRIBUTING.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/ISSUE_TEMPLATE/bug_report.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/ISSUE_TEMPLATE/config.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/ISSUE_TEMPLATE/feature_request.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/markdownlint.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/PULL_REQUEST_TEMPLATE.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/workflows/branch.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/workflows/ci.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/workflows/linting_comment.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/workflows/linting.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `assets/email_template.html`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `assets/email_template.txt`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `assets/sendmail_template.txt`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `assets/nf-core-smrnaseq_logo.png`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `bin/markdown_to_html.py`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `docs/images/nf-core-smrnaseq_logo.png`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `docs/output.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `docs/README.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `docs/README.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `docs/usage.md`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `lib/nfcore_external_java_deps.jar`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `lib/NfcoreSchema.groovy`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `main.nf`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `environment.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `Dockerfile`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `conf/base.config`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/workflows/awstest.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File found: `.github/workflows/awsfulltest.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found check: `Singularity`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found check: `parameters.settings.json`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found check: `bin/markdown_to_html.r`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found check: `conf/aws.config`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found check: `.github/workflows/push_dockerhub.yml`
* [files_exist](https://nf-co.re/tools-docs/lint_tests/files_exist.html) - File not found check: `.travis.yml`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `manifest.name`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `manifest.nextflowVersion`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `manifest.description`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `manifest.version`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `manifest.homePage`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `timeline.enabled`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `trace.enabled`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `report.enabled`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `dag.enabled`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `process.cpus`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `process.memory`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `process.time`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `params.outdir`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `params.input`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `manifest.mainScript`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `timeline.file`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `trace.file`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `report.file`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `dag.file`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable found: `process.container`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable (correctly) not found: `params.version`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable (correctly) not found: `params.nf_required_version`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable (correctly) not found: `params.container`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable (correctly) not found: `params.singleEnd`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable (correctly) not found: `params.igenomesIgnore`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable (correctly) not found: `params.name`
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``timeline.enabled`` had correct value: ``true``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``report.enabled`` had correct value: ``true``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``trace.enabled`` had correct value: ``true``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``dag.enabled`` had correct value: ``true``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``manifest.name`` began with ``nf-core/``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable ``manifest.homePage`` began with https://github.com/nf-core/
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``dag.file`` ended with ``.svg``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config variable ``manifest.nextflowVersion`` started with >= or !>=
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``process.container`` looks correct: ``nfcore/smrnaseq:dev``
* [nextflow_config](https://nf-co.re/tools-docs/lint_tests/nextflow_config.html) - Config ``manifest.version`` ends in ``dev``: ``'1.1dev'``
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.gitattributes` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `CODE_OF_CONDUCT.md` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `LICENSE` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/.dockstore.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/CONTRIBUTING.md` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/ISSUE_TEMPLATE/bug_report.md` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/ISSUE_TEMPLATE/config.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/ISSUE_TEMPLATE/feature_request.md` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/markdownlint.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/PULL_REQUEST_TEMPLATE.md` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/workflows/branch.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/workflows/linting_comment.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/workflows/linting.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `assets/email_template.html` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `assets/email_template.txt` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `assets/sendmail_template.txt` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `assets/nf-core-smrnaseq_logo.png` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `bin/markdown_to_html.py` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `docs/images/nf-core-smrnaseq_logo.png` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `docs/README.md` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `lib/nfcore_external_java_deps.jar` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `lib/NfcoreSchema.groovy` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.gitignore` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/workflows/push_dockerhub_dev.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `.github/workflows/push_dockerhub_release.yml` matches the template
* [files_unchanged](https://nf-co.re/tools-docs/lint_tests/files_unchanged.html) - `assets/multiqc_config.yaml` matches the template
* [actions_ci](https://nf-co.re/tools-docs/lint_tests/actions_ci.html) - '.github/workflows/ci.yml' is triggered on expected events
* [actions_ci](https://nf-co.re/tools-docs/lint_tests/actions_ci.html) - CI is building the correct docker image: `docker build --no-cache . -t nfcore/smrnaseq:dev`
* [actions_ci](https://nf-co.re/tools-docs/lint_tests/actions_ci.html) - CI is pulling the correct docker image: docker pull nfcore/smrnaseq:dev
* [actions_ci](https://nf-co.re/tools-docs/lint_tests/actions_ci.html) - CI is tagging docker image correctly: docker tag nfcore/smrnaseq:dev nfcore/smrnaseq:dev
* [actions_ci](https://nf-co.re/tools-docs/lint_tests/actions_ci.html) - '.github/workflows/ci.yml' checks minimum NF version
* [actions_awstest](https://nf-co.re/tools-docs/lint_tests/actions_awstest.html) - '.github/workflows/awstest.yml' is triggered correctly
* [actions_awsfulltest](https://nf-co.re/tools-docs/lint_tests/actions_awsfulltest.html) - `.github/workflows/awsfulltest.yml` is triggered correctly
* [readme](https://nf-co.re/tools-docs/lint_tests/readme.html) - README Nextflow minimum version badge matched config. Badge: `20.04.0`, Config: `20.04.0`
* [readme](https://nf-co.re/tools-docs/lint_tests/readme.html) - README had a bioconda badge
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda environment name was correct (nf-core-smrnaseq-1.1dev)
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::r-base=3.6.3`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::r-statmod=1.4.32`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::r-data.table=1.12.4`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::r-gplots=3.0.1.1`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::r-r.methodss3=1.7.1`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::python=3.7.3`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::markdown=3.1.1`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::pymdown-extensions=6.0`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::pygments=2.6.1`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `conda-forge::pigz=2.3.4`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::fastqc=0.11.9`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda package is the latest available: `bioconda::fastqc=0.11.9`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::trim-galore=0.6.5`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::samtools=1.9`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::bowtie=1.2.3`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::multiqc=1.9`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::mirtop=0.4.23`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda package is the latest available: `bioconda::mirtop=0.4.23`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::seqcluster=1.2.7`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda package is the latest available: `bioconda::seqcluster=1.2.7`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::htseq=0.11.3`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::fastx_toolkit=0.0.14`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda package is the latest available: `bioconda::fastx_toolkit=0.0.14`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::seqkit=0.12.0`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::mirtrace=1.0.1`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda package is the latest available: `bioconda::mirtrace=1.0.1`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::bioconductor-edger=3.26.5`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::bioconductor-limma=3.40.2`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda dep had pinned version number: `bioconda::mirdeep2=2.0.1.2`
* [conda_env_yaml](https://nf-co.re/tools-docs/lint_tests/conda_env_yaml.html) - Conda package is the latest available: `bioconda::mirdeep2=2.0.1.2`
* [conda_dockerfile](https://nf-co.re/tools-docs/lint_tests/conda_dockerfile.html) - Found all expected strings in Dockerfile file
* [pipeline_name_conventions](https://nf-co.re/tools-docs/lint_tests/pipeline_name_conventions.html) - Name adheres to nf-core convention
* [template_strings](https://nf-co.re/tools-docs/lint_tests/template_strings.html) - Did not find any Jinja template strings (56 files)
* [schema_lint](https://nf-co.re/tools-docs/lint_tests/schema_lint.html) - Schema lint passed
* [schema_lint](https://nf-co.re/tools-docs/lint_tests/schema_lint.html) - Schema title + description lint passed
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: linting.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: branch.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: awstest.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: awsfulltest.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: push_dockerhub_dev.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: ci.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: linting_comment.yml
* [actions_schema_validation](https://nf-co.re/tools-docs/lint_tests/actions_schema_validation.html) - Workflow validation passed: push_dockerhub_release.yml
* [merge_markers](https://nf-co.re/tools-docs/lint_tests/merge_markers.html) - No merge markers found in pipeline files

### Run details

* nf-core/tools version 1.13.3
* Run at `2021-04-13 08:50:21`

</details>

**Comment by ewels** [comment]
Looks like you have a few linting warnings that can be cleaned up still ☝🏻  The `pipeline_todos` and `schema_params` ones might warrant some attention (some stuff that hasn't been done can be left, eg. the aws full test stuff).

**Comment by ErikDanielsson** [comment]
I get no log messages when running the linting locally, oddly enough, @ewels.

**Comment by ewels** [comment]
> I get no log messages when running the linting locally, oddly enough, @ewels.

I suspect that this is due to a change in a dependency then - this stuff sometimes appears on GitHub Actions first as it does a totally fresh install every time so uses the latest versions of everything.

If we download the GHA artefact with the verbose log it might say which python module these messages are coming from..

**Comment by ewels** [comment]
Ah yes, our friend `requests_cache` doing loads of `log.info()` calls. Fantastic.

If you fancy a slightly more tricky / python task, a PR to nf-core/tools to downgrade these messages to debug would be fantastic.

Should be something like this ([see elsewhere](https://github.com/nf-core/tools/blob/03ab820b166b695ff40ebc9fa3cd53565a48577e/nf_core/lint/conda_env_yaml.py#L14-L16)):

```python
# Only show error messages from pipeline creation
logging.getLogger("requests_cache").setLevel(logging.WARNING)
```

**Comment by ErikDanielsson** [comment]
Sounds good, I'll give that a go!

**Comment by ewels** [comment]
> Ah yes, our friend `requests_cache` doing loads of `log.info()` calls. Fantastic.

Next time we're on a call, remind me to show you how I figured this out...

**Comment by ErikDanielsson** [comment]
I noticed that the schema param `config_profile_name` which is causing a lint warning is missing from the template nextflow.config as well, perhaps this test should warn with the current template?

**Comment by ewels** [comment]
> config_profile_name

Hmm I'm lost by this. x-ref https://github.com/nf-core/tools/pull/892/files . @KevinMenden do you know what should be going on here?

**Comment by KevinMenden** [comment]
Nope, no idea .... `config_profile_name` is nowhere to be found in the `nextflow.config` or `main.nf` .... so I'm wondering if we can just leave it out? 🤔

**Comment by KevinMenden** [comment]
Ah, you added that param about a month ago 🤔  okay I'm really lost here 🤷

**Comment by JWCook** [comment]
> Ah yes, our friend requests_cache doing loads of log.info() calls. Fantastic.

Those have been bumped down to debug level in requests-cache 0.6.1. You should have less spammy logs now.

**Comment by ewels** [comment]
> > Ah yes, our friend requests_cache doing loads of log.info() calls. Fantastic.
> 
> Those have been bumped down to debug level in requests-cache 0.6.1. You should have less spammy logs now.

Eagle eyes @jwcook! :octocat:  👀  That's great to hear, thanks for letting us know 👍🏻  Ok then I don't think any action is needed.. Even better 🎉

**Comment by ewels** [comment]
> I noticed that the schema param `config_profile_name` which is causing a lint warning is missing from the template nextflow.config as well, perhaps this test should warn with the current template?

@ErikDanielsson anyway yes, I think that `config_profile_name` should be in both the schema and `nextflow.config` 👍🏻 

I think that it also needs adding to the `nextflow.config` file in the template itself: https://github.com/nf-core/tools/blob/ee90d06f4e09c395c913a50b4266dd12fe7f179b/nf_core/pipeline-template/nextflow.config#L34 (as you spotted).

No change to the lint tests is needed.

**Comment by KevinMenden** [comment]
@ErikDanielsson are you still working on this?
Need to fix some bugs with the smRNA-seq pipeline, would be nice to merge this before :grin: 

I'll go through the PR either today or tomorrow to see if I can help.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['20.04.0', '']
+        nxf_ver: ['20.04.0', '21.03.0-edge']
```
Comment (KevinMenden) [inline .github/workflows/ci.yml:23]: can remove that again now

**Review by KevinMenden — COMMENTED** [review]
Alright this looks good to me, not sure why the tests are not running though :thinking:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ewels) [reply .github/workflows/ci.yml:23]:
```suggestion
        nxf_ver: ['20.04.0', '']
```

**Review by ewels — APPROVED** [review]
LGTM 🚀 

Thanks @ErikDanielsson for this, and thanks @KevinMenden for pushing it over the finishing line..

**Pull request merged**